File: TELOSTEL-F/telostel-frontend/src/app/model/carrito.ts
import { Categoria } from "./categoria"

export class Carrito {

    idProducto:number
    producto:string
    categoria:Categoria
    precio:number
    cantidad:number
    importe:number

    constructor(idProducto:number, producto:string,categoria:Categoria,precio:number,cantidad:number,importe:number){
        this.idProducto=idProducto
        this.producto=producto
        this.categoria=categoria
        this.precio=precio
        this.cantidad=cantidad
        this.importe=importe
    }

}

File: TELOSTEL-B/telostel-backend/src/main/java/com/dawii/dto/Carrito.java
package com.dawii.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Carrito {
	
	private Long idProducto;
	private String producto;
	private String idCategoria;
	private double precio;
	private int cantidad;
	private double importe;
	
}
